pkgs/by-name/ni/nixos-render-docs/src/nixos_render_docs/redirects.py

```diff
@@ -47,23 +47,49 @@ def __str__(self):
 
     If you moved content, add its new location as the first element of the redirects mapping.
     Please update doc/redirects.json or nixos/doc/manual/redirects.json!
-""") # TODO: automatically detect if you just missed adding a new location, and make a tool to do that for you
+""")
         if self.identifiers_without_redirects:
             error_messages.append(f"""
 Identifiers present in the source must have a mapping in the redirects file.
     - {"\n    - ".join(self.identifiers_without_redirects)}
 
     This can happen when an identifier was added or renamed.
-    Please update doc/redirects.json or nixos/doc/manual/redirects.json!
-""") # TODO: add tooling in the development shell to do that automatically and point to that command
+
+    Added new content?
+        redirects add-content <identifier> <path>
```

Comment:
The `<idenfier> <path>` doesn't show in the browser output when running changes with` devmode`